lib/zoho_hub/base_record.rb

```diff
@@ -241,4 +250,13 @@ def build_response(body)
       self.class.build_response(body)
     end
   end
+
+  class Note < BaseRecord
```

Comment:
An idea to lighten this huge class: create a module, like we did [for the attachments](https://github.com/Fabien-Wecasa/zoho_hub/pull/5/files), put in the `Note` class and related method (`notes` and `add_note`),  and `require` it in `zoho_hub.rb`.